pkg/test/memory.go

```diff
@@ -249,6 +260,18 @@ func TXTRegisterSpaceValid(txtAPI hwapi.APIInterfaces) (bool, error, error) {
 	return true, nil, nil
 }
 
+// TXTPublicReservedInE820 checks if TXTPublic area is marked reserved in e820 map
+func TXTPublicReservedInE820(txtAPI hwapi.APIInterfaces) (bool, error, error) {
+	res, err := txtAPI.IsReservedInE820(uint64(tools.TxtPublicSpace), uint64(tools.TxtPublicSpace+0x10000))
```

Comment:
Is it always 0x10000? Is it an offset?

Reply:
Document Number: 558294, Revision: 2.0, page 48